modules/otel-configuring-otelcol-metrics.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * otel/otel-configuring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-otelcol-metrics_{context}"]
+= Configuring the OpenTelemetry Collector metrics
+
+You can enable metrics and alerts of OpenTelemetry Collector instances.
+
+.Prerequisites
+
+* Monitoring for user-defined projects is enabled in the cluster. See xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[Enabling monitoring for user-defined projects].
```

Comment:
Can't xref from modules (works on the preview but breaks on the RH customer portal)

Reply:
This has already been addressed since it was mentioned in a previous comment.